scripts/apps/authoring/views/authoring-topbar.html

```diff
@@ -13,11 +13,10 @@
     ></sd-authoring-topbar-react>
 
     <div ng-show="isLocked()" class="locked-info">
-      <figure class="avatar small">
-          <div class="user-image">
-              <img sd-user-avatar data-user="item.lock_user">
-          </div>
-      </figure>
+
+      <div style="margin-right: 4px; margin-left: -2px; transform: scale(0.9);">
```

Comment:
Why inline styles? Also, why isn't there a prop for the avatar sizes? We have at least 3 sizes